content/dlgnewedit.xul

```diff
@@ -109,6 +109,17 @@
 
 <vbox id="advSection" hidden="true">
 <groupbox>
+    <caption label="&requestMethod.label;" />
+    <radiogroup id="requestMethod" onselect="UpdateScanner.DlgNewEdit.requestMethodChanged();">
+        <radio label="GET" selected="true"/>
+        <hbox align="center">
+            <radio label="POST"/>
+            <textbox id="postParams" size="50" disabled="True"/>
```

Comment:
It's not immediately obvious what this textbox needs to contain. Maybe add a placeholder attribute with an example value as a guide?